text.md

```diff
@@ -269,6 +269,15 @@ The following flags are supported:
 * `--smtp_password=abc123` Specify the SMTP server password.
 * `--imap_password=abc123` Specify the IMAP server password.
 
+## Environment variables
+
+The following environment variables are supported:

```

Comment:
Maybe mention the evaluation order/priority, or specifically that the config value must be absent or empty for these to be used?

Reply:
Good point, I explained password priority in 3335a93